client/go/cmd/testdata/tests/expected-suite.out

```diff
@@ -1,7 +1,9 @@
-Running testdata/tests/system-test/test.json: .... OK!
-Running testdata/tests/system-test/wrong-bool-value.json:
-Failed verifying assertion 0:
-Expected JSON at /root/coverage/full (false) does not match actual (true). Response body:
+Running my test: .... OK
```

Comment:
Could you add another successful test right after this?  I just want to verify there is no empty line between two successful tests.

Reply:
Hmm, there is an empty line between all tests. Would you rather it weren't?

Reply:
It's nice with an empty line after an unsuccessful test, because the error message for the test is itself multiline, so it would be difficult to see where the next test begins.

Reply:
But between successful tests there should be no empty lines? You want

```
Running foo.json: ....ok
Running bar.json: ....ok
Running zoo.json: ....ok
```